bounties_api/std_bounties/management/commands/bounties_subscriber.py

```diff
@@ -125,14 +132,6 @@ def handle_message(self, message):
             defaults=event_arguments
         )
 
-        # This means the contract subscriber will never send this event
```

Comment:
No call to `self.remove_from_queue(message)` here - we want to do that each individual location: 
- If a message has been blacklisted but there's a new event for it (never reaches `handle_message`)
- After a regular message has been handled successfully
- When the event throws and fails